api/acl.go

```diff
@@ -750,7 +750,7 @@ func (a *ACL) TokenUpdate(token *ACLToken, q *WriteOptions) (*ACLToken, *WriteMe
 // of an existing token.
 func (a *ACL) TokenClone(tokenID string, description string, q *WriteOptions) (*ACLToken, *WriteMeta, error) {
```

Comment:
Should we rename `tokenID` to `accessorID` here?

Reply:
done